packages/cli/src/ui/commands/chatCommand.test.ts

```diff
@@ -185,30 +185,32 @@ describe('chatCommand', () => {
       });
     });
 
-    it('should inform if conversation history is empty', async () => {
+    it('should inform if conversation history is empty or only contains system context', async () => {
       mockGetHistory.mockReturnValue([]);
-      const result = await saveCommand?.action?.(mockContext, tag);
+      let result = await saveCommand?.action?.(mockContext, tag);
```

Comment:
nit: rather than reusing the same variable can you create new ones. result1, result2, etc